src/console_implementation/drawable_player.cpp

```diff
@@ -1,7 +1,7 @@
 #include "drawable_player.hpp"
 #include "../business_logic/include/Tposition.hpp"
 
-drawable_player::drawable_player(WINDOW* gameplay_window, std::string init_name) : m_gameplay_window(gameplay_window), Tplayer(init_name)
+drawable_player::drawable_player(WINDOW* gameplay_window, std::string init_name) : Tplayer(init_name), m_gameplay_window(gameplay_window)
```

Comment:
Consider using smart pointers - RAII.
```suggestion
drawable_player::drawable_player(std::shared_ptr<WINDOW> gameplay_window, std::string init_name) : Tplayer(init_name), m_gameplay_window(gameplay_window)
```